changelog/2663.breaking.rst

```diff
@@ -0,0 +1,3 @@
+Removal of `InteractiveVSOClient`, `Attr.Wave`, `GenericClient.get`,
```

Comment:
Should say these were deprecated. Also according to the changelog readme removal of deprecated things is a `removal` type not a `breaking` type.